plugins/UserCountry/lang/en.json

```diff
@@ -1,51 +1,51 @@
-        "GeolocationProviderUnavailable": "The configured geolocation provider %1$s is not available anymore, please configure another one.",
-        "getCityDocumentation": "This report shows the cities your visitors were in when they accessed your website.",
-        "getContinentDocumentation": "This report shows which continent your visitors were in when they accessed your website.",
-        "getCountryDocumentation": "This report shows which country your visitors were in when they accessed your website.",
-        "getRegionDocumentation": "This report shows which region your visitors were in when they accessed your website.",
+        "GeolocationPageDesc": "Here you can change how Matomo determines visitor locations.",
+        "GeolocationProviderBroken": "Your geolocation provider (%1$s) is broken. Please fix the provider, or set up another one.",
+        "GeolocationProviderUnavailable": "Your geolocation provider (%1$s) is no longer available. Please set up another one.",
+        "getCityDocumentation": "Shows the cities your visitors connected from when accessing your website.",
+        "getContinentDocumentation": "Shows which continent your visitors connected from when accessing your website.",
+        "getCountryDocumentation": "Shows which country your visitors connected from when accessing your website.",
+        "getRegionDocumentation": "Shows which region your visitors connected from when accessing website.",
         "HowToInstallGeoIPDatabases": "How do I get geolocation databases?",
         "Latitude": "Latitude",
         "Latitudes": "Latitudes",
         "Location": "Location",
-        "LocationProvider": "Location Provider",
+        "LocationProvider": "Location provider",
         "Longitude": "Longitude",
         "Longitudes": "Longitudes",
-        "NoDataForGeoIPReport1": "There is no data for this report because there is either no location data available or visitor IP addresses cannot be geolocated.",
-        "NoDataForGeoIPReport2": "To enable accurate geolocation, change the settings %1$shere%2$s and use a %3$scity level database%4$s.",
-        "NoProviders": "There are currently no additional geolocation providers available. Matomo recommends using the %1$sdbip%2$s databases, but this requires activating the GeoIp2 plugin. (Alternatively you can install a third party plugin that provides it's own geolocation functionality from the marketplace.)",
-        "PluginDescription": "Reports the location of your visitors: country, region, city and geographic coordinates (latitude\/longitude).",
+        "NoDataForGeoIPReport1": "No data for this report, as there is either no location data available, or visitor IP addresses cannot be geolocated.",
+        "NoDataForGeoIPReport2": "%1$Change these settings%2$s to get accurate geolocation, and a %3$scity-level database%4$s.",
```

Comment:
The user gets both, so not needed.

Reply:
The user actually needs to change the geolocation provider settings and needs to explicitly use a city level database by either using a downloader that is implemented or uploading the database manually. So that's nothing that will automatically happen by changing the settings.